docs/conf/eu/2016/news/thanks-for-coming.md

```diff
@@ -0,0 +1,64 @@
+:template: 2016/eu.html
+
+Thanks for coming to Write the Docs Europe 2016
+===============================================
+
+Hi everyone,
+
+Thanks for attending our third European Write the Docs conference in Prague! We're so grateful to our sponsors, speakers, volunteers, and attendees for making the conference possible. 
```

Comment:
I'd feel better if we didn't write sponsors first ;) so something like "speakers, volunteers, sponsors, and attendees"